lib/ecl/tests/ecl_smspec.cpp

```diff
@@ -23,28 +23,13 @@
 #include <ert/ecl/ecl_sum.hpp>
 #include <ert/ecl/ecl_smspec.hpp>
 
-void test_sort( ecl_smspec_type * smspec )
```

Comment:
Note: this test is removed because the whole `ecl_smspec_sort()` functionality is removed.